packages/blockchain-info-components/src/Form/SelectInput/index.js

```diff
@@ -57,8 +61,8 @@ class SelectInputContainer extends React.PureComponent {
         handleChange={this.handleChange}
         searchEnabled={this.props.searchEnabled}
         grouped={grouped}
-        {...rest}
         onBlur={this.onBlur}
+        {...rest}
```

Comment:
this change's breaking blur behavior

Reply:
@tony-blockchain passing `{...rest}` is breaking? Looks like the cWRP --> cDU transition wasn't done properly. That's been fixed now.